app/controllers/api_controller/reports.rb

```diff
@@ -1,12 +1,20 @@
 class ApiController
   module Reports
+    SCHEDULE_ATTR = %w(start_date interval time_zone send_email).freeze
```

Comment:
It is user's API side schedule attribute that is use for create MiqSchedule's attribute like `sched_action` or `run_at`. WDYT about SCHEDULE_BODY_ATTR ?